### ISSUE TITLE
Some images of the gtav dataset are dropped during testing.

### ISSUE THREAD

**Issue by xyIsHere** [shachoi/RobustNet#5]
Dear authors,

Thank you for your great work and I learned a lot of new stuffs from this work. I just have a small question about the use of gtav dataset. I saw some images from the gtav dataset are dropped during testing, so are the reported mIoU for gtav is only based on part of the validation set? (I know the comparisons are still fair.)

And the test set (6,181 images) of the gtav is not used in this work, right?

**Comment by shachoi** [comment]
Yes, That's right.
The test set was not used, the model was trained on the training set, and the test was performed on the validation set.
Also, some images from GTAV were dropped during testing.
As mentioned in the README, I split the GTAV data as shown in [this link](https://github.com/shachoi/RobustNet#we-used-gtav_split-to-split-gtav-dataset-into-trainingvalidationtest-set-please-refer-the-txt-files-in-split_data)

**Comment by xyIsHere** [comment]
Thank you for your quick response.
